libs/chlorophyll/scss/components/accordion/_mixins.scss

```diff
@@ -89,4 +90,20 @@
       }
     }
   }
+
 }
+
+@mixin accordion-list-item-content() {
+  overflow: hidden;
+  transition: height 200ms linear, opacity 200ms linear;
+  margin-left: 1em;
+
+  &.content-hidden {
+      opacity: 0;
+      visibility:hidden;
+  }
+
+  &:not(.content-hidden) {
```

Comment:
We left this padding out since it created so much space between label and content. On hover it's very close between button and content, but dicussed with designers and we chose to skip it.

Reply:
Removed.